io.seamly.seamly2d.yml

```diff
@@ -0,0 +1,62 @@
+app-id: io.seamly.seamly2d
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+command: seamly2d
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=cups
+  - --filesystem=home
+  - --device=dri
+  - --socket=pulseaudio
+rename-desktop-file: seamly2d.desktop
+separate-locales: false
```

Comment:
Why disable this?

Reply:
The translation files are embedded as Qt resources, thats why I (also in the previous package) disabled the generation of an empty locale extension